app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/Windows.java

```diff
@@ -275,6 +289,14 @@ private WindowWidget addRestoredWindow(@NonNull WindowState aState, @NonNull Ses
         newWindow.getPlacement().worldWidth = aState.worldWidth;
         newWindow.setRestored(true);
         placeWindow(newWindow, aState.placement);
+        switch (aState.panelType) {
```

Comment:
I'm getting a crash here:
https://crash-stats.mozilla.com/report/index/ded91d9d-9f44-46ba-bc9c-18fca0200123
https://crash-stats.mozilla.com/report/index/e8e5b8d8-e61a-4b3a-87e9-9770f0200123
STR:
1) Run FxR without this patch applied and quit so windows are saved.
2) Apply patch and re-run FxR
FxR will crash on start up.

Reply:
Good catch. I've added a check to see if the new panel information is present.